ftml/misc/ftml-base.css

```diff
@@ -1,156 +1,248 @@
 /* ELEMENTS */
 
 .wj-image-container {
-    padding: 1em;
+  padding: 1em;
 }
 
 .wj-email {
-    white-space: nowrap;
+  white-space: nowrap;
 }
 
 .wj-raw {
-    white-space: pre-wrap;
+  white-space: pre-wrap;
 }
 
 /* TABLES */
 
-.wj-table {}
+.wj-table {
+}
 
 /* FOOTNOTES */
 
 a.wj-footnote-ref {
-    display: inline-block;
+  display: inline-block;
 }
 
-.wj-footnotes-list {}
+.wj-footnotes-list {
+}
 .wj-footnotes-list .wj-title {
-    font-size: 1.3em;
-    font-weight: bold;
+  font-size: 1.3em;
+  font-weight: bold;
 }
 
 .wj-footnotes-list ol {
-    list-style: none;
+  list-style: none;
 }
 
-.wj-footnotes-list .wj-footnote {}
-.wj-footnotes-list .wj-footnote span.wj-footnote-sep {}
-.wj-footnotes-list .wj-footnote div.wj-footnote-contents {}
+.wj-footnotes-list .wj-footnote {
+}
+.wj-footnotes-list .wj-footnote span.wj-footnote-sep {
+}
+.wj-footnotes-list .wj-footnote div.wj-footnote-contents {
+}
 
 /* COLLAPSIBLE */
 
-.wj-collapsible-block {}
-.wj-collapsible-block-link {}
-.wj-collapsible-block-unfolded-link {}
+.wj-collapsible-block {
+}
+.wj-collapsible-block-link {
+}
+.wj-collapsible-block-unfolded-link {
+}
 
 .wj-collapsible-block-content {
-    display: block;
+  display: block;
 }
 
 .wj-collapsible-block-folded {
-    display: none;
+  display: none;
 }
 
 .wj-collapsible-block-unfolded {
-    display: block;
+  display: block;
 }
 
 /* CODE */
 
-code,
 .wj-code {
-    font-family: 'Courier New', Courier, monospace;
-    padding: 0 1em;
-    margin: 0.4em 0;
-    overflow: auto;
+  border: 1px solid #ccc;
+  border-radius: 0.25em;
+  padding: 0.5em 1em;
+  hyphens: none;
+  line-height: 1.5;
+  text-align: start;
+  word-break: normal;
+  word-wrap: normal;
+  tab-size: 4;
+  white-space: pre;
+  cursor: text;
+  contain: content;
+}
+
+.wj-code code {
+  display: block;
+  font-family: "Courier New", Courier, monospace;
```

Comment:
Should we use a different monospace font here?

Reply:
I'm actually fine with Courier New, I forgot I was thinking of just plain Courier (which is what `monospace` is, as well) when I said the default font was ugly.

I could make a quick sorta-decent-native-font-stack but I think this is okay. This is one of those things that every site should try to customize, I think.